.github/workflows/pr_agent.yml

```diff
@@ -17,5 +17,8 @@ jobs:
         id: pragent
         uses: Codium-ai/pr-agent@main
         env:
-          OPENAI_KEY: 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
-          GITHUB_TOKEN: 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
+          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
```

Comment:
It's recommended to validate the existence and correctness of the secrets before using them. This can prevent potential issues if the secrets are not set or incorrect. [medium]